src/styles/getModalStyles.js

```diff
@@ -120,7 +120,7 @@ export default (type, windowDimensions) => {
                 overflow: 'hidden',
             };
 
-            swipeDirection = 'right';
+            swipeDirection = undefined;
```

Comment:
Maybe let's leave a comment here as a hint in case someone decides they want a gesture and then breaks everything?

Reply:
Done!